docs/content/templates/functions.md

```diff
@@ -460,6 +460,22 @@ e.g.
 
 * `{{ int "123" }}` → 123
 
+### lang.NumFmt
+
+`NumFmt` formats a number with a given precision using the requested *decimal*,
+*grouping*, and *negative* characters.  The function signature is
+`NumFmt <decimal> <grouping> <negative> <precision> <number>`.
+
+Note that numbers are rounded up at 5 or greater.
+So, with precision set to 0, 1.5 becomes `2`, and 1.4 becomes `1`.
+
+```
+{{ lang.NumFmt "," "." "-" 2 12345.6789 }} → 12.345,68
```

Comment:
We have no concept of locale or "locale formatting". What you are saying is that we should wait until we see how that looks like before we revisit this one.